src/codeu/chat/ClientMain.java

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
```

Comment:
minor note: you probably don't want to commit "no op" changes like this, since it just clutters up the pull request